plone/namedfile/tests/test_scaling.py

```diff
@@ -63,6 +63,7 @@ def testCreateScale(self):
         foo = self.scaling.scale('image', width=100, height=80)
         self.assertTrue(foo.uid)
         self.assertEqual(foo.mimetype, 'image/jpeg')
+        self.assertEqual(foo.data.contentType, 'image/jpeg')
```

Comment:
these tests need to be enhances as:

```python
>>> u'image/jpeg' == 'image/jpeg'
True
```

we need to guarantee it's an instance of `str`